ot/__init__.py

```diff
@@ -1,81 +0,0 @@
-"""
```

Comment:
This file should not have been deleted!

please pit it again an complete it (the documentation about the submodule), also iumport the new submodule by default.